src/dotnet/Notification.Service/Db/DbDevice.cs

```diff
@@ -0,0 +1,38 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using ActualChat.Notification.Backend;
+using Microsoft.EntityFrameworkCore;
+using Stl.Versioning;
+
+namespace ActualChat.Notification.Db;
+
+[Table("Devices")]
+[Index(nameof(UserId))]
+public class DbDevice : IHasId<string>, IHasVersion<long>
+{
+    private DateTime _createdAt;
+    private DateTime _accessedAt;
+
+    public DbDevice() { }
+
+    [Key] public string Id { get; set; } = null!;
```

Comment:
is it intentional that an attribute with default property value goes in one line with property and for the other cases it goes on a separate line

Reply:
The difference is caused by our auto formatting rules - e.g. Short attributes can be placed on the same line